### PR TITLE
Add note in Prerequisites to upgrade EL version for 3.3

### PR DESCRIPTION
**Pull request by bangelic**
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc [hunk 1]
```diff
@@ -37,6 +37,11 @@ For more information, see {InstallingServerDocURL}Preparing_your_Environment_for
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 
+ifdef::satellite[]
+NOTE: {Project} {ProjectVersion} is released only on {EL} 8. You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
+For more information on upgrading from {EL} 7 to {EL} 8, see xref:upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}[].
```
Comment (adamlazik1) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc:42]:
```suggestion
NOTE: {Project} {ProjectVersion} is released only on {EL} 8.
You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
For more information on upgrading from {EL} 7 to {EL} 8, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/upgrading_and_updating_red_hat_satellite/upgrading-satellite-or-proxy-in-place-using-leapp_upgrade-guide[Upgrading Satellite or Capsule to {RHEL} 8 In-Place Using Leapp].
```
[Ewoud's comment](https://github.com/theforeman/foreman-documentation/pull/1720#discussion_r1010600337) is indeed correct. If there are no procedures to upgrade from el 7 to 8 for satellite 6.12 then a link for 6.11 has to be included. @evgeni or @ekohl please confirm these changes.

File: guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc [hunk 2]
```diff
@@ -37,6 +37,12 @@ For more information, see {InstallingServerDocURL}Preparing_your_Environment_for
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 
+ifdef::satellite[]
+NOTE: {Project} {ProjectVersion} is released only on {EL} 8.
+You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
+For more information about upgrading from {EL} 7 to {EL} 8, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/upgrading_and_updating_red_hat_satellite/upgrading-satellite-or-proxy-in-place-using-leapp_upgrade-guide[Upgrading Satellite or Capsule to {RHEL} 8 In-Place Using Leapp].
```
Comment (ekohl) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc:43]: Would it make sense to reuse the version macro? And call out that it's the manual section for 6.11?
```suggestion
For more information about upgrading from {EL} 7 to {EL} 8, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/upgrading-satellite-or-proxy-in-place-using-leapp_upgrade-guide[Upgrading Satellite or Capsule {ProjectVersionPrevious} to {RHEL} 8 In-Place Using Leapp].
```

File: guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc [hunk 3]
```diff
@@ -37,6 +37,12 @@ For more information, see {InstallingServerDocURL}Preparing_your_Environment_for
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 
+ifdef::satellite[]
+NOTE: {Project} {ProjectVersion} is released only on {EL} 8.
+You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
```
Comment (evgeni) [inline guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc:42]:
```suggestion
If you're running {Project} {ProjectVersionPrevious} on {EL} 7, you must first upgrade to {EL} 8 before you can continue with the upgrade to {Project} {ProjectVersion}.
```
(or something along these lines?)

Comment (evgeni) [reply guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc:42]: (but this is me being picky, wouldn't block on the wording here)

**Review by evgeni — APPROVED** [review]
Ack on the content.

I guess you want this squashed and picked to 3.3+?

**Comment by bangelic** [comment]
> Ack on the content.
> 
> I guess you want this squashed and picked to 3.3+?

Yes please!

**Comment by evgeni** [comment]
given this was already open against 3.3, no picks were necessary, I should read better.

squashed and merged

**Pull request merged**